packages/cli/src/commands/pipeline-deploy/pipeline_deploy_command.ts

```diff
@@ -79,6 +83,12 @@ export class PipelineDeployCommand
         type: 'string',
         array: false,
       })
+      .option('out-dir', {
+        describe:
+          'A path to directory where config is written. If not provided defaults to current process working directory.',
+        type: 'string',
+        array: false,
+      })
```

Comment:
I think we should be consistent with `sandbox` command here and use this more explicit naming https://github.com/aws-amplify/amplify-backend/blob/6a2f9aa8fe81bf48ac725470a1bd64622966da46/packages/cli/src/commands/sandbox/sandbox_command.ts#L143-L156 
The shorter naming is fine in `generate config` as it's easier to reason what that arg is there. But apparently there was some in `sandbox` hence longer name there.
And also consider adding second parameter - format as well.

Notice that we didn't find a good way to reuse parameters across commands (mixins?), so I recommend to just copy from sandbox.